capstan.go

```diff
@@ -503,7 +503,7 @@ func main() {
 					Usage: "collects contents of this package and all required packages",
 					Flags: []cli.Flag{
 						cli.BoolFlag{Name: "pull-missing, p", Usage: "attempt to pull packages missing from a local repository"},
-						cli.StringFlag{Name: "boot", Usage: "specify config_set name to boot unikernel with"},
+						cli.StringFlag{Name: "boot", Usage: "specify config_set name to boot unikernel with (repeatable, will be run left to right)"},
```

Comment:
You've removed the `boot` below entirely. Why do you keep it here then?

Is it ok to remove the `boot` from `CollectPackage`?